include/mata/nfa/nfa.hh

```diff
@@ -402,51 +442,6 @@ Nfa intersection(const Nfa& lhs, const Nfa& rhs,
 Nfa concatenate(const Nfa& lhs, const Nfa& rhs, bool use_epsilon = false,
```

Comment:
concatenation, this creates a new nfa, right?

Reply:
Yes. We can rename it to `concatenation()`. Anyone else any opinions on this?